### DISCUSSION TITLE
v4.15.0

### DISCUSSION THREAD

**Discussion by marcelklehr** [floccusaddon/floccus#1187] [General]
### New
- [Native] AddBookmarkIntent: Autodetect page title
- NewAccount: Allow setting enabled account config
- NewAccount: Allow setting XBEL passphrase for GoogleDrive and WebDAV

### Fixed
- Fix order corruption of localRoot folder
- Tabs: Fix syncing multiple windows
- NewAccount: Warn user when using server without https
- Improve UI so there's space for translations
- NewAccount: Remove stepper headings so the whole stepper fits
- Failsafe: added Math.ceil to only allow integers
- New translations for Polish, French and Chinese

<hr /><em>This discussion was created from the release <a href='https://github.com/floccusaddon/floccus/releases/tag/v4.15.0'>v4.15.0</a>.</em>

**Comment by pinpontitit** [comment]
Was waiting for this release. Now the app is perfect ! ❤️

**Reply by akwala** [reply]
Noticed the wording change in the dropdown from the toolbar button. It is clear and concise. Would be good to use the same wording under Synchronization Strategy on the Options page. I'd avoid the word 'revert', [it can be confusing](https://en.wiktionary.org/wiki/revert).

Minor: I suggest listing the three options, Push, Pull and Merge, in the same order in both places, with the default/recommended one as the first.

**Reply by pinpontitit** [reply]
I agree with you.

I noticed that the "sync strategy choice" is available in two different location for the browser extension:
in the main page, and in the option page.

I think both of them are used for the same purpose but, if they arent, please correct me.

We could erase one of them instead of keeping both.

Or, as you said, if we keep both, and if they actually are used for the same purpose, we should use exactly the same words in the same order for both, to avoid confusion.
If not, users could think "ah ok this is a different option than the one in the main  page".

I'm not english but if "revert" is confusing, I think something like ~~"replace"~~ "cancel" could be quite clear, what do you think ?

**Reply by akwala** [reply]
> I think both of them are used for the same purpose but, if they arent, please correct me.
>
> We could erase one of them instead of keeping both.

Yes. If they have the same purpose then there should should be a single place to set this. If they are not the same then that's obviously not clear.

**Reply by pinpontitit** [reply]
I definitely think they serve same purpose because: 
if you select pull in the main page, it changes to pull in option. If you change to merge in main page, it changes to merge in option.

I had never tried so I wasnt sure.

**Reply by pinpontitit** [reply]
And I made mistake. Revert could be replaced by "cancel".

For example:
"always cancel (or undo) local changes and replace by remote changes"

Anyway.. let's stop off-topic and open an enchancement issue if we want those changes :stuck_out_tongue:

**Reply by marcelklehr** [reply]
I like undo. Adjusted the wording and removed the setting from the options page for browser version.

**Reply by akwala** [reply]
What it says here is just right, IMO (the description needs to wrap so that all of it is visible – it's fully visible onhover):
![image](https://user-images.githubusercontent.com/1786/171944952-759f6bfa-8775-469e-816e-c11b7286066e.png)

**Reply by marcelklehr** [reply]
Yep, that'll stay the way it is. Wrapping the description is tough with that component, though, I'm already glad I got it to show on hover.
